Updated SVG Sprite Icon Organization

grayghostvisuals opened the pull request.

ryelle commented on Gruntfile.js:
```diff
@@ -602,8 +601,8 @@ module.exports = function(grunt) {
 			dev: {
-				dest: BUILD_DIR + 'wp-includes/fonts/dashicons.svg',
-				src: [ SOURCE_DIR + 'wp-includes/fonts/.tmp/*.svg' ]
+				dest: BUILD_DIR + 'wp-includes/icons/src/dashicons.svg',
```
This shouldn't be built into the `src` directory, `src` is just for the collection of individual icons.

grayghostvisuals commented:
- replaced the icon font files that were removed
- adjusted the original svg files to live in an icons/lib folder
- adjusted the grunt task to build the sprite into the icons/src dir when executing build task
